packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -202,6 +233,7 @@ class MockRuntime
 			await uploadP;
 			this.processing = false;
 		}
+		await new Promise<void>((resolve) => setTimeout(resolve, delay));
```

Comment:
Is this just to make the TTL expire?

Reply:
Correct

Reply:
So this covers the scenario where an offline blob expires between upload and submitting the op, without disconnect. I don't really expect this to happen, but it's fine to cover this edge case.

The main scenario we want to test is if a blob is uploaded, then the user disconnects long enough for the blob to expire, then reconnects. I think that should be possible to test with the mock runtime as it is.